setup_julia.py

```diff
@@ -1,9 +1,6 @@
 import juliapkg
 
-
 if __name__ == "__main__":
     juliapkg.require_julia("1.8")
-    juliapkg.add("Jabalizer", "5ba14d91-d028-496b-b148-c0fbc366f709", version="0.4.3")
-    juliapkg.add("JSON", "682c06a0-de6a-54ab-a142-c8b1cf79cde6", version="0.21.3")
-
+    juliapkg.add("Jabalizer", "5ba14d91-d028-496b-b148-c0fbc366f709", version="0.4")
```

Comment:
Is this the right version?

Reply:
But the changes that you just introduced are not available in 0.4.3, they are only 0.4.4 up. So perhaps you can introduce some ">"  bounds or sth like that?

CC: @SebastianMorawiec regarding what it looks at, cause he implemented that part.